### PR TITLE
Update esi.rst

### PR DESCRIPTION
**Pull request by atailouloute**
Fixing some portions of PHP code that are not surrounded with PHP tags

### CODE REVIEW

File: http_cache/esi.rst [hunk 1]
```diff
@@ -137,9 +137,10 @@ matter), Symfony uses the standard ``render`` helper to configure ESI tags:
     .. code-block:: html+php
 
         <!-- app/Resources/views/static/about.html.php -->
-
+        <?php
         // you can use a controller reference
         use Symfony\Component\HttpKernel\Controller\ControllerReference;
+        ?>
         <?php echo $view['actions']->render(
```
Comment (Nek-) [inline http_cache/esi.rst]: It's useless to have end/open blocks.

**Review by Nek- — CHANGES_REQUESTED** [review]

File: http_cache/esi.rst (hunk 1 above)
Comment (javiereguiluz) [inline http_cache/esi.rst]: It's "useless" most of the times ... but in this case this is an example of a PHP template, so you must open/close PHP tags whenever you add PHP code. However, I don't know which is the current practice in the docs regarding the `use ...` imports. Maybe we do an exception for those blocks? Let's ask to @wouterj and @xabbuh. Thanks!

**Review by roman-1983 — APPROVED** [review]

File: http_cache/esi.rst (hunk 1 above)
Comment (atailouloute) [inline http_cache/esi.rst]: In other template code examples, each piece of PHP code is surrounded by open/close tags as @javiereguiluz said? I just did the same and I am not sure about what's the correct practice in this case

**Review by HeahDude — APPROVED** [review]
👍 Code seems currently broken. This change looks good to me.

**Comment by xabbuh** [comment]
Thank you for the many good catches and improvements @atailouloute! To ease the merging of all the small fixes I will combine your PRs in a single PR (see #7737) and merge that one instead.

**Pull request closed without merging**